dev/tests/integration/testsuite/Magento/Directory/Controller/Adminhtml/Json/CountryRegionTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Directory\Controller\Adminhtml\Json;
+
+use Magento\Framework\App\Request\Http as HttpRequest;
+use Magento\TestFramework\TestCase\AbstractBackendController;
+
+/**
+ * @magentoAppArea adminhtml
+ */
+class CountryRegionTest extends AbstractBackendController
+{
+    /**
+     * Test Execute without param
+     */
+    public function testExecuteWithNoCountryParam()
+    {
+        $this->getRequest()->setMethod(HttpRequest::METHOD_POST);
+        $this->getRequest()->setPostValue([]);
+        $this->dispatch('backend/directory/json/countryRegion');
+
+        $actual = $this->getResponse()->getBody();
+
+        $this->assertEquals('[]', $actual);
+    }
+    /**
+     * Test Execute with param = Australia
+     */
+    public function testExecuteWithCountryAustralia()
+    {
+        $this->getRequest()->setMethod(HttpRequest::METHOD_POST);
+        $this->getRequest()->setPostValue([
+            'parent' => 'AU'
+        ]);
+        $this->dispatch('backend/directory/json/countryRegion');
+
+        $australiaRegionArray = [
+            [
+                'title' => '',
+                'value' => '',
+                'label' => 'Please select a region, state or province.'
+            ],
+            [
+                'value' => '569',
```

Comment:
I'm not sure it's a good idea to rely on the sequence number. It can lead to unstable results on different environments.
Could you please simplify the assertion and let it be less strict?